cmd/protoc-gen-connect-go/main.go

```diff
@@ -225,6 +211,24 @@ func generateServiceNameConstants(g *protogen.GeneratedFile, services []*protoge
 	g.P()
 }
 
+func generateServiceNameVariables(g *protogen.GeneratedFile, file *protogen.File) {
+	wrapComments(g, "These variables are the protoreflect.Descriptor objects for the ", file.Desc.Name(),
+		" service's methods.")
```

Comment:
This comment doesn't seem too helpful. It's a little confusing because `file.Desc.Name()` returns the last element of the package.

Did you mean `file.Desc.Path()`? That would actually be awkward, too. Also, grammatically this probably wants to say "services and methods" instead of "service's methods" because there could be multiple services and "services' methods" is probably a little confusing to read.

Reply:
Thanks for catching, simplified to:
```
// These variables are the protoreflect.Descriptor objects for the RPCs defined in this package.
```